docs/integrations/nbext.md

```diff
@@ -1,47 +1,53 @@
 # Jupyter Notebook Extension
 
 Now you can commit your Jupyter Notebook with to [Jovian](https://jovian.ml?utm_source=docs) just **One Click**.
-Make sure you’ve completed the [Installation](../user-guide/01-install.md) before reading further.
+Make sure you’ve completed the [Installation](https://jovian-py.readthedocs.io/en/latest/user-guide/01-install.html) before reading further.
 
 ## Using Jovian Jupyter Extension
 
-<img src="https://i.imgur.com/RdBnJYy.png" class="screenshot" alt="jupyter extension toolbar button">
+<img src="https://i.imgur.com/44QTHP6.png" class="screenshot" alt="jupyter extension toolbar button">
```

Comment:
Both are similar images, why was this replaced?

Reply:
The different is the arrow button (dropdown menu button).